components/clp-package-utils/clp_package_utils/scripts/decompress.py

```diff
@@ -76,6 +75,13 @@ def main(argv):
         return -1
     extraction_dir.mkdir(exist_ok=True)
 
+    # Validate and load config file
+    clp_config = validate_and_load_config(
+        clp_home, pathlib.Path(parsed_args.config), default_config_file_path
+    )
+    if not clp_config:
```

Comment:
Since the return type of `validate_and_load_config` is an `Optional[]`, would it be better to check the return value against `None`?
```suggestion
    if clp_config is not None:
```

Reply:
I guess you intended to write `if clp_config is None:`?

I am fine with the change, but we do have many occurence of code where we return an Optional and not explitilcitly check it against None (iirc). so imo maybe we should first decide if this should be a standard.

Reply:
Sorry yeah I meant `if clp_config is None:`. Sure, let's hold the changes until me (or someone else) formally reviews this PR.

Reply:
@kirkrodrigues any suggestions?

Reply:
I think the best practice is not to treat None as a boolean. So explicit `is None` and `is not None` is better.